search/templates/search/search.html

```diff
@@ -107,7 +114,8 @@
         for (var i=0; i < markerList.length; i++){
             var location_url = markerList[i][2];
             var image_url = markerList[i][3];
-            content = `<img width="100" src="${image_url}"/><br><a target="_blank" href="${location_url}">View Location</a>`;
+            var price = markerList[i][4]
+            content = `<img width="100" src="${image_url}"/><br><h6 style="color: green" > ${price} </h6><br><a target="_blank" href="${location_url}" >View Location</a><br>`;
```

Comment:
nit: would it be possible to reduce the spacing between the price and the view location link just a bit? Otherwise looks great

Reply:
Yes, I was about to point this out.

Reply:
@ab7289 @Ayushi-0903 Updated!